### PR TITLE
Updated postman collection

### PR DESCRIPTION
**Pull request by drupeshUnwired**

### CODE REVIEW

File: LocationIQ Collection v2.0.0.postman_collection.json [hunk 1]
```diff
@@ -1022,7 +1058,7 @@
 								{
 									"key": "countrycodes",
 									"value": "us",
-									"description": "Limit search to a specific of country.",
+									"description": "Limit search to a specific of country. Now supports multiple countrycodes. For example : \"us,ca\"",
```
Comment (gopi-ar) [inline LocationIQ Collection v2.0.0.postman_collection.json]: Limit search to a list of countries. Link to the notes section of docs for more info. Or specify examples inline.

File: LocationIQ Collection v2.0.0.postman_collection.json [hunk 2]
```diff
@@ -1040,13 +1076,19 @@
...
+									"description": "Restricts the autocomplete search results to elements of specific OSM class and type.  Now supports multiple tags. Examples :-\n1. To return only cities: tag=place:city\n2. To return only types of place, such as Suburbs, Towns and Cities, use a wildcard: tag=place:*\n3. To restrict results to specific types of place: tag=place:city,place:town,place:village\n4. To restrict results to cafes: tag=amenity:cafe",
+									"disabled": true
+								},
+								{
+									"key": "dedupe",
+									"value": "0",
+									"description": "Sometimes you have several objects in OSM identifying the same place or object in reality. The simplest case is a street being split in many different OSM ways due to different characteristics. Our Geocoder will attempt to detect such duplicates and only return one match; this is controlled by the dedupe parameter which defaults to 1. Since the limit is, for reasons of efficiency, enforced before and not after de-duplicating, it is possible that de-duplicating leaves you with less results than requested.",
```
Comment (gopi-ar) [inline LocationIQ Collection v2.0.0.postman_collection.json]: dedupe=0 by default for Autocomplete for b/c.

File: LocationIQ Collection v2.0.0.postman_collection.json [hunk 3]
```diff
@@ -1100,8 +1142,8 @@
 								},
 								{
 									"key": "countrycodes",
-									"value": "in",
-									"description": "Limit search to a specific of country."
+									"value": "fr",
+									"description": "Limit search to a specific of country. Now supports multiple countrycodes. For example : \"us,ca\""
```
Comment (gopi-ar) [inline LocationIQ Collection v2.0.0.postman_collection.json]: Same as above

File: LocationIQ Collection v2.0.0.postman_collection.json [hunk 4]
```diff
@@ -1118,13 +1160,19 @@
...
+									"description": "Restricts the autocomplete search results to elements of specific OSM class and type.  Now supports multiple tags. Examples :-\n1. To return only cities: tag=place:city\n2. To return only types of place, such as Suburbs, Towns and Cities, use a wildcard: tag=place:*\n3. To restrict results to specific types of place: tag=place:city,place:town,place:village\n4. To restrict results to cafes: tag=amenity:cafe",
+									"disabled": true
+								},
+								{
+									"key": "dedupe",
+									"value": "0",
+									"description": "Sometimes you have several objects in OSM identifying the same place or object in reality. The simplest case is a street being split in many different OSM ways due to different characteristics. Our Geocoder will attempt to detect such duplicates and only return one match; this is controlled by the dedupe parameter which defaults to 1. Since the limit is, for reasons of efficiency, enforced before and not after de-duplicating, it is possible that de-duplicating leaves you with less results than requested.",
```
Comment (gopi-ar) [inline LocationIQ Collection v2.0.0.postman_collection.json]: 0

File: README.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ We recommend using REST clients like <a href="https://www.postman.com" target="_
 ### Load Collection
 
 1. Download the provided <br/>
-    - <a href="https://raw.githubusercontent.com/location-iq/postman-collection/master/LocationIQ_Collection_v1.1.0.postman_collection.json" target="_blank">LocationIQ Collection</a> file (`Right-Click -> Save As`). 
+    - <a href="https://raw.githubusercontent.com/location-iq/postman-collection/master/LocationIQ_Collection_v2.0.0.postman_collection.json" target="_blank">LocationIQ Collection</a> file (`Right-Click -> Save As`).
```
Comment (gopi-ar) [inline README.md]: We should stick to LIQ's major version - v1. You can increment the minor ver instead.

File: LocationIQ Collection v2.0.0.postman_collection.json (hunk 1 above)
Comment (drupeshUnwired) [reply LocationIQ Collection v2.0.0.postman_collection.json]: made changes

File: LocationIQ Collection v2.0.0.postman_collection.json (hunk 2 above)
Comment (drupeshUnwired) [reply LocationIQ Collection v2.0.0.postman_collection.json]: added info about default value

File: LocationIQ Collection v2.0.0.postman_collection.json (hunk 4 above)
Comment (drupeshUnwired) [reply LocationIQ Collection v2.0.0.postman_collection.json]: All the integer value also comes in quote after exporting it from Postman.

File: README.md (hunk 5 above)
Comment (drupeshUnwired) [reply README.md]: Changed the changed version to 1.3.0 since 2 new API's, Timezone and Optimize were added in the previous collection.

**Comment by rocainunwired** [comment]
Overriden by https://github.com/location-iq/postman-collection/pull/8

**Pull request closed without merging**